.github/bin/label_abandoned_prs.py

```diff
@@ -7,45 +7,45 @@
 terminal_output = subprocess.check_output(pr_json, shell=True, text=True)
 json_output = json.loads(terminal_output)
 
-inactive_comment = "This PR has been inactive (no commits and no review comments)"\
-    " for 12 days. If there is no new activity in the next 48 hours,"\
-    " this PR will be labeled as Abandoned PR - Needs New Owner and"\
-    " either another developer will finish the PR or it will be closed."
-
 eastern = datetime.timezone(datetime.timedelta(hours=-5))
 today = datetime.datetime.now()
 two_weeks = timedelta(weeks=2)
-twelve_days = timedelta(days=12)
-two_days = timedelta(days=2)
 et_today = today.astimezone(eastern)
 
 for json_data in json_output:
-    already_warned = False
-    updated_at_string = json_data['updatedAt']
-    if json_data["comments"] and json_data["comments"][-1]["body"] == inactive_comment:
-        already_warned = True
-
-    json_time = datetime.datetime.fromisoformat(updated_at_string.replace('Z', '+00:00'))
+    updated_at_string = json_data["updatedAt"]
+    json_time = datetime.datetime.fromisoformat(
+        updated_at_string.replace("Z", "+00:00")
+    )
     et_time_update = json_time.astimezone(eastern)
 
     tdiff = et_today - et_time_update
+    num = str(json_data["number"])
 
-    num = str(json_data['number'])
     already_abandoned = False
-    for labels in json_data['labels']:
-        if labels['name'] == 'Abandoned PR - Needs New Owner':
+    for labels in json_data["labels"]:
+        if labels["name"] == "Abandoned PR - Needs New Owner":
             already_abandoned = True
 
     approved = False
-    for review in json_data['reviews']:
+    for review in json_data["reviews"]:
```

Comment:
Please limit this PR to only the most minimal set of changes necessary.  If you wish to make formatting changes, please make them in a separate PR so formatting changes don't hide logic changes.

Reply:
Ah oops. I ran an auto-formatter on that, I can revert that.

Reply:
Reverted.